src/status_im/contexts/wallet/common/token_value/view.cljs

```diff
@@ -44,11 +43,18 @@
                           (rf/dispatch [:wallet/bridge-select-token bridge-params]))})
 
 (defn- action-swap
-  []
-  {:icon                :i/swap
-   :accessibility-label :swap
-   :label               (i18n/label :t/swap)
-   :on-press            #(not-implemented/alert)})
+  [token-symbol]
+  (let [current-viewing-account-address (rf/sub [:wallet/current-viewing-account-address])
+        account-address                 (or current-viewing-account-address
+                                            (:address (first (rf/sub [:wallet/operable-accounts]))))
```

Comment:
Hey @ajayesivan !

Better to create a separate subscription that handles this logic.
The `:operable-accounts` doesn't specify an order and we are taking the first one here, could you please elaborate on it?